website/meta/universe.json

```diff
@@ -2943,6 +2943,7 @@
                 "from pysbd.utils import PySBDFactory",
                 "",
                 "nlp = spacy.blank('en')",
+                "# works with spaCy<=2.x.x",
```

Comment:
It would be better to annotate this with the `spacy_version` meta data as well (this is a new feature that ideally we'd add for all the entries). Maybe better to do that in a separate PR?